online-devops-dojo/welcome/intro.md

```diff
@@ -6,14 +6,15 @@ Welcome. I am the Octocat for the **Online DevOps Dojo** Katacoda course
 
 You are about to enter another dimension, a dimension not only of sight and sound but of mind. A journey into a wondrous land of imagination. Next stop, the **Online DevOps Dojo**!
 
-This course is made up of 6 open source modules.
+This course is made up of 7 open source modules.
 
 * Leading Change
```

Comment:
Just cross referencing with https://github.com/dxc-technology/about-devops-dojo/pull/64, the announcement suggests 7 modules with this post incident practices being the newest No 8.  This list says 6 current + 1 new or 7 total.  Looks like the difference is the announcement says Welcome and Setup is the first module.

Just suggest to align the two?

Reply:
@adamdmharvey again good point, change made.